jecs.luau

```diff
@@ -1158,7 +1158,7 @@ do
 				local idr_t_archetype = archetypes[archetype_id]
 
 				local idr_t_types = idr_t_archetype.types
-				local on_remove = idr_t.hooks.on_remove
+				
```

Comment:
I am the last person to comment on formatting and style but let\s remove this whitespace